src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -321,7 +322,7 @@ export const Overview: React.FC<OverviewProps> = ({
               mt='15px'
               mb='8px'
             />
-            {isConnected && !isDemoWallet ? (
+            {isConnected && accountId && !isDemoWallet ? (
```

Comment:
Revert me, else this will never run, the `onChange()` callback on l.339 will never run, and `accountId` will never be set, resulting in this:

<img width="491" alt="image" src="https://github.com/user-attachments/assets/a164b513-616e-4b6b-9c15-da395f4612b4">